### DISCUSSION TITLE
Real time chat

### DISCUSSION THREAD

**Discussion by jo-so** [pinnacle-comp/pinnacle#183] [General]
Is there any kind of real time chat system (Matrix/IRC/Discord) to ask questions/talk about pinnacle?

**Comment by Ottatop** [comment]
Not yet, I'll make a Discord server sometime later today

**Comment by ildar** [comment]
maybe in the Smithay matrix room? I think Smithay people may be interested
in Pinnacle.

**Reply by Ottatop** [reply]
Not really sure how I feel about piggybacking off of their Matrix. I think a dedicated room/space is a better solution.

**Comment by Ottatop** [comment]
Aight links to the Discord and Matrix are at the top of the README, hopefully they work
